tests/Support/Constraint/StringContainsCssCount.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\Tests\Support\Constraint;
+
+use PHPUnit\Framework\Constraint\Constraint;
+
+/**
+ * This constraint asserts that the string it is evaluated for contains some specific CSS a specific number of times,
+ * allowing for cosmetic whitespace differences.
+ *
+ * The CSS string and expected number of occurrences are passed in the constructor.
+ *
+ * @author Jake Hotson <jake.github@qzdesign.co.uk>
+ */
+class StringContainsCssCount extends Constraint
+{
+    use CssConstraint;
+
+    /**
+     * @var int
+     */
+    private $count;
+
+    /**
+     * @var string
+     */
+    private $css;
+
+    /**
+     * @param int $count
+     * @param string $css
+     */
+    public function __construct(int $count, string $css)
+    {
+        parent::__construct();
+
+        $this->count = $count;
+        $this->css = $css;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function toString(): string
+    {
+        return 'contains exactly ' . (string)$this->count . ' occurrence(s) of CSS `' . $this->css . '`';
```

Comment:
Any particular reason why you are using backticks here and no double quotes?

Reply:
Because it's code (it's CSS, but still code 😉).  I note that in the PHPUnit tests they use `"`, but those are for generic strings.  Quotes are for what people said.  Backticks are for what people coded.  I think.  YMMV...

Reply:
The backtick thing comes from Markdown as it is used on GitHub, and then spilled over to some PHPDoc comments. I've never seen it applied to strings in code before. (I'm not opposed to this, either.)

Reply:
> The backtick thing comes from Markdown

Actually I think its use in Markdown comes from historical usage for quoting code.  Note that PHP has a [backtick operator](https://www.php.net/manual/en/language.operators.execution.php) that, I'm sure, predates Markdown.